### PR TITLE
update affiliation for Diego

### PR DESCRIPTION
**Pull request by diekus**
***
<a href="https://pr-preview.s3.amazonaws.com/diekus/screen-fold/pull/52.html" title="Last updated on May 4, 2021, 8:17 AM UTC (6a78596)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/device-posture/52/d88b2b3...diekus:6a78596.html" title="Last updated on May 4, 2021, 8:17 AM UTC (6a78596)">Diff</a>

### CODE REVIEW

**Comment by kenchris** [comment]
I think we just say, previously at Samsung. @anssiko ?

**Comment by anssiko** [comment]
@diekus first needs to re-join with his new affiliation per instructions at https://www.w3.org/das/participate

(The contributions incorporated in the FPWD were from the time when @diekus was affiliated with Samsung.)

**Comment by anssiko** [comment]
With @diekus on board now, we can merge this, assuming @diekus is still willing to step up as a co-editor.

@kenchris don’t worry about the checks failing, Travis CI needs some fixing and ipr check is confused because we renamed the repo in between.

**Comment by diekus** [comment]
Sounds about right. More than happy to continue as editor. 😊

File: index.html [hunk 1]
```diff
@@ -15,8 +15,8 @@
         editors: [{
           name: "Diego González",
           url: "https://twitter.com/diekus",
-          company: "Samsung",
-          companyURL: "https://developer.samsung.com/internet",
+          company: "Microsoft [Samsung until 2020]",
```
Comment (anssiko) [inline index.html]:
```suggestion
          company: "Microsoft, formerly on behalf of Samsung",
```

**Review by anssiko — APPROVED** [review]

**Comment by anssiko** [comment]
Welcome back @diekus!

**Pull request merged**